keps/prod-readiness/sig-storage/3301.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3301
```

Comment:
Just to clarify, would the KEP number be **`4876`** since this is related to GitHub k/enhancement issue https://github.com/kubernetes/enhancements/issues/4876? If so, we’ll need to update the KEP number in the PR files as well as the filenames to ensure they align with **_KEP-4876_**. Apologies for any confusion on my part.

Reply:
Fixed.